app/views/content/what-is-qts.md

```diff
@@ -19,12 +20,25 @@ keywords:
   - State-school
 ---
 
-Qualified teacher status (QTS) is what you need to teach in a state school in England. 
+Qualified teacher status (QTS) is what you need to teach in most primary and secondary schools in England.
 
-It is a confirmation that you have the right training and meet the standards required to be a teacher. 
+It is a confirmation that you have the right training and meet the standards required to be a teacher.
 
-All postgraduate teacher training courses in England lead to QTS. You can see the different types of courses available by [searching for postgraduate teacher training](https://www.gov.uk/find-postgraduate-teacher-training-courses).
+## How to get QTS
+
+All postgraduate teacher training courses in England lead to QTS. You can see the different types of courses available by [searching for postgraduate teacher training]
```

Comment:
this link is not rendering properly (because it is on separate lines here??)

![image](https://user-images.githubusercontent.com/87642394/149196904-e49206aa-4beb-4f2f-854d-e1b395cf9255.png)